tests/old-ui-test/README.md

```diff
@@ -46,7 +46,7 @@ This virtual machine will be the FreeNAS target running ui tests with firefox.
 
 
 #Download freenas/webui repo
```

Comment:
`freenas` -> `truenas`

Reply:
Thanks! Fixed in f51dd4f14596e3514c5f524878f9ff3ce9e1400f